src/test/java/apiStepsDefs/ApiStepsDefs.java

```diff
@@ -0,0 +1,55 @@
+package apiStepsDefs;
+
+import io.cucumber.java.en.*;
+
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Cookie;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import utils.ApiHelper;
+
+import java.util.concurrent.TimeUnit;
+
+import static constants.Constants.IMPLICITLY_WAIT_TIMEOUT;
+import static driver.CapabilitiesHelper.getChromeOptions;
+
+public class ApiStepsDefs {
+
+    ApiHelper apiHelper = new ApiHelper();
+    String baseUrl = "https://www.kruidvat.nl";
+
+    @Given("create a cart")
+    public void createACart(){
+        apiHelper.createCart(baseUrl);
```

Comment:
base url should be set globally, so you wouldn't need to pass it to each api method as an arg